core/src/test/java/org/elasticsearch/action/fieldcaps/FieldCapabilitiesTests.java

```diff
@@ -106,4 +109,75 @@ static FieldCapabilities randomFieldCaps() {
             randomAlphaOfLengthBetween(5, 20), randomBoolean(), randomBoolean(),
             indices, nonSearchableIndices, nonAggregatableIndices);
     }
+
+    @Override
+    protected MutateFunction<FieldCapabilities> getMutateFunction() {
```

Comment:
Good point, I've changed this to `protected T mutateInstance(T instance)` in the tests